cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3596,6 +3596,142 @@ action.cyences_notable_event_action.attacker_drilldown = `cs_linux_sudousers` $r
+| eval users=case(status=="group_added",new_users,status=="group_removed",old_users,status=="group_updated",new_users) \
+| table host group_name _time users status \
+| outputlookup cs_linux_groups append=true
+
+[Linux - cs_linux_users Lookup Gen]
+disabled = 0
+enableSched = 1
+alert.track = 0
+cron_schedule = 7 * * * *
+description = 
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.timeRangePicker.show = 0
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+display.visualizations.show = 0
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_linux_userdetails` UID!=""\
+| eval new_COMMAND_SHELL = COMMAND_SHELL, new_GID=GID, new_HOME_DIR=HOME_DIR,new_SUDOACCESS=SUDOACCESS,new_USER_INFO=USER_INFO,new_USERNAME=USERNAME\
+| stats latest(new_*) as new_* latest(_time) as new_time by host UID \
+| eval host_status = "reporting" \
+| appendpipe \
+    [| inputlookup cs_linux_users \
+    | eval old_COMMAND_SHELL = COMMAND_SHELL, old_GID=GID, old_HOME_DIR=HOME_DIR,old_SUDOACCESS=SUDOACCESS,old_USER_INFO=USER_INFO,old_USERNAME=USERNAME\
+    | stats latest(old_*) as old_* latest(status) as status latest(_time) as old_time by host UID \
+    | search status!="removed" \
+    | fields - status ] \
+| stats values(*) as * by  host UID \
+| eventstats values(host_status) as host_status , max(new_time) as host_latest by host \
+| eval status=case(isnull(old_USERNAME) AND host_status=="reporting","user_added",isnull(new_USERNAME) AND host_status=="reporting","user_removed",(new_COMMAND_SHELL!=old_COMMAND_SHELL OR new_HOME_DIR!=old_HOME_DIR OR new_SUDOACCESS!=old_SUDOACCESS OR new_USER_INFO!=old_USER_INFO OR new_GID!=old_GID OR new_USERNAME!=old_USERNAME ) AND host_status=="reporting","user_updated",1=1,"no change") \
+| search status!="no change" \
+| eval _time=case(status=="user_added",new_time,status=="user_removed",host_latest,status=="user_updated",new_time) \
+| eval USERNAME=case(status=="user_added",new_USERNAME,status=="user_removed",old_USERNAME,status=="user_updated",new_USERNAME) \
+| eval COMMAND_SHELL=case(status=="user_added",new_COMMAND_SHELL,status=="user_removed",old_COMMAND_SHELL,status=="user_updated",new_COMMAND_SHELL) \
+| eval HOME_DIR=case(status=="user_added",new_HOME_DIR,status=="user_removed",old_HOME_DIR,status=="user_updated",new_HOME_DIR) \
+| eval SUDOACCESS=case(status=="user_added",new_SUDOACCESS,status=="user_removed",old_SUDOACCESS,status=="user_updated",new_SUDOACCESS) \
+| eval USER_INFO=case(status=="user_added",new_USER_INFO,status=="user_removed",old_USER_INFO,status=="user_updated",new_USER_INFO) \
+| eval GID=case(status=="user_added",new_GID,status=="user_removed",old_GID,status=="user_updated",new_GID) \
+| table host UID _time USERNAME COMMAND_SHELL HOME_DIR SUDOACCESS USER_INFO GID status \
+| outputlookup cs_linux_users append=true
+
+[Linux - User Added/Updated/Deleted]
```

Comment:
Can we simply the name? 
* Linux - User Changes

Reply:
For dashboard panel as well